src/main/java/com/gdsc/pikpet/controller/TestController.java

```diff
@@ -1,13 +1,44 @@
 package com.gdsc.pikpet.controller;
 
+import com.google.auth.oauth2.GoogleCredentials;
+import com.google.cloud.storage.*;
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.RestController;
 
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.nio.file.Paths;
+
 @RestController
 public class TestController {
 
     @GetMapping("/")
     public String test() {
         return "Hello World!";
     }
+
+    @GetMapping("/testImage")
+    public String testImage() throws IOException {
+        String bucketName = "solution-challenge-bucket"; // Google Cloud Storage 버킷 이름
+        String imageName = "good.jpg"; // 업로드할 이미지 파일 이름
```

Comment:
```suggestion
    // Google Cloud Storage 버킷 이름
    @Value("${application.bucket.name}") 
    private String bucketName;

    @GetMapping("/")
    public String test() {
        return "Hello World!";
    }

    @GetMapping("/testImage")
    public String testImage() throws IOException {
        String imageName = "good.jpg"; // 업로드할 이미지 파일 이름
```
버킷이름도 secret하게 관리하시죠!